dt-posts/custom-post-type.php

```diff
@@ -58,9 +57,10 @@ public function register_post_type(){
             'labels'                => $labels,
             'public'                => false,
             'publicly_queryable'    => true,
-            'show_ui'               => true,
-            'show_in_menu'          => true,
+            'show_ui'               => false,
+            'show_in_menu'          => false,
```

Comment:
This removes the link from the WP > admin which we still want

Reply:
Do we still want visibility of contacts, groups, and other post types in the admin? Why?

Reply:
The notifications and title active background color changes didn't show up on my system. I'll look for the bug on this.

Reply:
Reversed the admin hide. But think we should consider it. https://github.com/DiscipleTools/disciple-tools-theme/pull/1203/commits/4085ad3c712090cbfbd36c6415e79672498629ee

Reply:
@ChrisChasm It is currently the only way to bulk delete records. A group needed this just this past week.